src/AutoRest.CSharp/Common/Input/CodeModelPartials.cs

```diff
@@ -20,6 +20,8 @@ internal partial class Operation
     {
         // For some reason, booleans in dictionaries are deserialized as string instead of bool.
         public bool IsLongRunning => Convert.ToBoolean(Extensions.GetValue<string>("x-ms-long-running-operation") ?? "false");
+
+        public bool IsKeepClientDefaultValue => Configuration.MethodsToKeepClientDefaultValue.Contains(OperationId);
```

Comment:
the name of this property reads weird. Maybe we change it to `ShouldKeepClientDefaultValue`? Or just `KeepClientDefaultValue`

Reply:
updated